.github/workflows/build.yml

```diff
@@ -0,0 +1,98 @@
+name: Build
+
+on: [push]
+
+jobs:
+  Linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y build-essential cmake libgl-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Build
+        run: |
+          cmake -B bin -DCMAKE_BUILD_TYPE=Release .
```

Comment:
```suggestion
          cmake -B bin -DCMAKE_BUILD_TYPE=Release -G Ninja .
```

Reply:
I added `--parallel 4` instead, which turns out to be equally fast as using Ninja.

Here a comparison:

- Parallel option: https://github.com/sharkwouter/OpenNFS/actions/runs/11840113594
- Ninja: https://github.com/sharkwouter/OpenNFS/actions/runs/11840176830